CHANGELOG.md

```diff
@@ -28,6 +29,1178 @@ under the License.
 - [1.4.2](#142-sat-mar-19-000806-2022-0200)
 - [1.4.1](#141)
 
+### 2.1 (Thu Mar 16 21:13:05 2023 -0700)
```

Comment:
Please use current master for any change/PR

Reply:
It looks like this is merging into master now.